src/TEdit/View/Popups/NewWorldView.xaml

```diff
@@ -26,7 +26,7 @@
             </DockPanel>
             <TextBlock Text="Height" HorizontalAlignment="Right" />
             <DockPanel>
-                <TextBlock Text="{Binding TilesHigh, StringFormat={}{0:0}}" Width="30" DockPanel.Dock="Right" />
+                <TextBlock Text="{Binding TilesHighReactive, StringFormat={}{0:0}}" Width="30" DockPanel.Dock="Right" />

```

Comment:
This text box and the slider are bound to different properties.

Reply:
Yes this is intended. This ensures that `TilesHigh` still controls its background UI and also updates `UpdateMaxLayerLevels();`. The slider is supposed to be controlling `TilesHigh` while its reactive is updating and reflecting back `TilesHighReactive`. This solves the issues mentioned above.